index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Tekle's Portfolio</title>
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"
+      integrity="sha384-Bfad6CLCknfcloXFOyFnlgtENryhrpZCe29RTifKEixXQZ38WheV+i/6YWSzkz3V"
+      crossorigin="anonymous"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <header class="toolbar">
+      <div class="container">
+        <p class="logo">My Logo</p>
+        <a href="#" class="btn-menu-toggle"><i class="fas fa-bars"></i></a>
+      </div>
+    </header>
+    <section class="headline">
+      <div class="container">
+        <h2 class="headline--title">I’m Tekle Glad to see you!</h2>
+        <p class="headline--body">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <p class="headline--supporting-text">Let's connect</p>
+        <div class="social-media">
+          <a href="" class="social-media-items twitter-link">
+            <img src="./img/icons/twitter_icon.svg" alt="twitter icon" />
+          </a>
+          <a href="" class="social-media-items linkedin-link">
+            <img src="./img/icons/Linkedin-icon.svg" alt="LinkedIn icon" />
+          </a>
+          <a href="" class="social-media-items gmail-link">
+            <img src="./img/icons/gmail_icon.svg" alt="Gmail icon" />
+          </a>
+          <a href="" class="social-media-items victory-link">
+            <img src="./img/icons/victory_icon.svg" alt="Victory icon" />
+          </a>
+        </div>
```

Comment:
- [ ] The list of icons should be grouped with the `<ul>` tag not the `<div>` tag and an `<li>` tag per icon.